spec/compiler/parser/to_s_spec.cr

```diff
@@ -749,4 +749,86 @@ describe "ASTNode#to_s" do
         .map do |v| v end
     %}
     CR
+
+  expect_to_s <<-'CR'
+    {%
+      (
+        1
+      )
+    %}
+    CR
+
+  expect_to_s <<-'CR'
+    {%
+      (
+        1
+        2
+      )
+    %}
+    CR
+
+  expect_to_s <<-'CR'
+    {%
+      (
+        true ||
+        false
```

Comment:
This should be indented the same as the regular code, I guess?

```suggestion
        true ||
          false
```

ditto all below

Reply:
Dunno, formatter doesn't do anything with it so imo it's fine as it is. The main objective was to have proper line numbers. Column numbers are less important. Esp when it's subjective what the indentation should be.

Reply:
I suppose the formatter really should act on code _inside_ macro expressions. Let's leave that for a follow-up, though.